src/index.jsx

```diff
@@ -9,22 +9,8 @@ import Engine from "./engine";
 
 (function main() {
   const { ship, organization, secret } = queryParams();
-
-  Hull.init({
-    appId: ship,
-    orgUrl: `https://${organization}`,
-    embed: false
-  });
-
-  Hull.ready((hull, currentUser, app) => {
-    const root = document.getElementById("app");
-    const engine = new Engine({ ship, organization, secret }, { ship: app, currentUser });
-
-    window.addEventListener("message", ({ data: message }) => {
-      if (message && message.event === "ship.update" && message.ship) {
-        engine.updateShip(message.ship);
-      }
-    });
-    ReactDOM.render(<App engine={engine} />, root);
-  });
+  const root = document.getElementById("app");
+  const userSearch = window.localStorage.getItem(`userSearch-${ship}`);
+  const engine = new Engine({ ship, organization, secret }, { ship: {}, userSearch });
```

Comment:
previously we were passing ship: app which was passed from hull.js

Are we sure passing an empty object is ok?  Don't we need the code from the ship to put it in the window?  Is it getting there another way now?

Reply:
Stephane: it’s rehydrated by the compute endpoint, here:

https://github.com/hull-ships/hull-processor/blob/develop/server/actions/compute-handler.js#L17

Reply:
it's already gracefull handled here : https://github.com/hull-ships/hull-processor/blob/develop/server/actions/compute-handler.js#L17

Reply:
Tested the branch, for the code pull in the editor, seems to work